db/schema.rb

```diff
@@ -11,16 +11,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150128222759) do
+ActiveRecord::Schema.define(version: 20150211221341) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
 
   create_table "guides", force: true do |t|
     t.string   "location"
     t.integer  "user_id"
-    t.datetime "created_at",                   null: false
-    t.datetime "updated_at",                   null: false
+    t.datetime "created_at",                                                               null: false
+    t.datetime "updated_at",                                                               null: false
     t.string   "specialty"
     t.float    "rate"
-    t.boolean  "availability", default: false
+    t.boolean  "availability", default: [false, false, false, false, false, false, false],              array: true
```

Comment:
This should not be here. See if you still have a file like "<timestamp>_add_availability_to_guide.rb" in your db/migrate folder. If you do delete it, if you don't, then this will get removed upon the next rake db:migrate:reset and it's not an issue.